Gemfile

```diff
@@ -29,6 +29,8 @@ group :test, :development do
   gem 'guard-teaspoon', '~> 0.0'
   gem 'guard-spork',    '~> 1.5'
   gem 'guard-rubocop',  '~> 1.0'
+  gem 'guard-bundler',  '~> 2.1.0'
```

Comment:
Vooral voor gems die niet op productie komen scheelt die laatste 0 weghalen een heleboel upgrade geneuzel. De lock zet dingen al vast voor dagelijks gebruik en mocht het ooit problemen opleveren zien we het meteen en zetten we het dan vast.